docs/discussion/selection-api.md

```diff
@@ -0,0 +1,378 @@
+  objectClicked      |Fires when an object is clicked | Catalog object and xy coordinates of the click event | Existing
+  footprintClicked   |Fires when a footprint is clicked | Footprint object xy coordinates of the click event | Existing
+  -----------------------------------------------------------------------------------
+
+## 3.2. GET
+
+#### GetSelectedSources
+```
+/**
+ * @returns List[Sources]
+ */
+function GetSelectedSources()
+```
+Returns the list of currently selected source objects.
+
+#### GetSelectedRegion
+```
+/**
+ * @returns Region
+ */
+function GetSelectedRegion()
+```
+Returns the region definition of the selection event, if a region was
+used to select. 
+
+#### GetSelectedFootprints
+```
+/**
+ *@returns List[Footprint]
+ */
+function GetSelectedFootprints()
+```
+Returns the currently selected footprints
+
+## 3.3. SELECT
+
+#### SelectSources 
+
+##### Option 1: 
+```
+/**
+ * @param sources: List[Source]
+ * @param keys: List[str]
+ */
+function SelectSources(sources, keys)
+```
+Selects the list of sources provided within aladin-lite
+
+  **Parameter**   |**Type**         |**Description**
+  --------------- |---------------- |--------------------------------------------
+  sources         |List[Source]     |The list of sources to select.
+  keys            |List[str]        |The list of keys to use to determine a match
+  -----------------------------------------------------------------------------
+
+##### Option 2:
+```
+/**
+ * @param match_func: func()
+ */
+function SelectSources(match_func)
+```
+Selects the list of sources provided. Uses the match_func provided to
+determine if a source should be selected
+
+  **Parameter**   |**Type**    |**Description**
+  --------------- |----------- |---------------------------------------------------
+  match_func      |func(a,b)   |The function used to determine if an object should be selected
+
+  -------------------------------------------------------------------------------
+
+#### SelectRegion
+```
+/**
+ * @param region: Region
+ */
+function SelectRegion(region)
+```
+Triggers a region selection within aladin-lite widget
+
+  **Parameter**             |**Type**          |**Description**
+  ------------------------- |----------------- |---------------------------
+  region                    |Region            |The region to select objects within
+
+  -----------------------------------------------------------------------
+
+#### SelectFootprint
+```
+/**
+ * @param footprint: Footprint
+ */
+function SelectFootprint(footprint)
+```
+Triggers a footprint selection within aladin-lite widget
+
+  **Parameter**       |**Type**         | **Description**
+  ------------------- |---------------- |-----------------------------------
+  footprint           |Footprint        |The footprint to select
+
+
+## 3.4. Misc
+
+#### RemoveSelection
```

Comment:
Is this meant to remove all selections including footprints and sources with possible multiple layers?

Reply:
I intended for this to be a pass through to the [unselectObjects](https://github.com/cds-astro/aladin-lite/blob/master/src/js/View.js#L1463) method in aladin-lite. My understanding is that is unselects both footprints and sources